test_runner/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -74,6 +74,8 @@ object Versions {
 
     // https://github.com/mockk/mockk
     const val MOCKK = "1.9.3"
+
+    const val COMMON_TEXT = "1.7"
```

Comment:
Please, add here link to repo or equivalent, thanks.

Reply:
Comment added